addons/project/views/project_views.xml

```diff
@@ -681,6 +681,7 @@
             <field name="model">project.project</field>
             <field name="arch" type="xml">
                 <tree decoration-muted="active == False" string="Projects" multi_edit="1" sample="1">
+                    <field name="company_id" invisible="1"/>
```

Comment:
Can you add the company_id field below the other `company_id` one? 
It is to keep selecting the right company_id with an eventual xpath. 🙂 

Thanks in advance! 🙂